data-prepper-plugins/opensearch/build.gradle

```diff
@@ -63,10 +51,6 @@ dependencies {
     implementation 'software.amazon.awssdk:url-connection-client:2.17.15'
     implementation 'software.amazon.awssdk:arns:2.17.15'
     implementation 'io.micrometer:micrometer-core:1.7.5'
-    // The OpenSearch build-tools plugin appears to be preventing Gradle's platform
-    // support from working correctly, so we have to specify the JUnit versions here.
-    testImplementation "org.junit.jupiter:junit-jupiter:${versionMap.junitJupiter}"
-    testImplementation "org.junit.vintage:junit-vintage-engine:${versionMap.junitJupiter}"
```

Comment:
It's always nice to see `org.junit.vintage` removed.

Reply:
Actually this project is still using it because it comes from the root Gradle project. And all the tests are still vintage JUnit tests.